modules/negotiate/lib/Auth/Source/Negotiate.php

```diff
@@ -66,6 +67,7 @@ public function __construct($info, $config)
         $this->admin_user = $config->getString('adminUser', null);
         $this->admin_pw = $config->getString('adminPassword', null);
         $this->attributes = $config->getArray('attributes', null);
+        $this->rcache = $config->getBoolean('rcache', null);
```

Comment:
I find the name of the config option a bit obscure, but that could also be due to my lack of knowledge on this 😓 

In any case, we should probably mention this new option (and its rationale) in the documentation.